docs/src/1_Mechanics/07_Items.yaml

```diff
@@ -0,0 +1,180 @@
+Template:               # must be unique
```

Comment:
Let's try to simplify down to exactly what we need for now. We could work on a sample version to see if that works when loaded as part of a PC render. A StatOverride feature could work here like i've implemented for Powers